app/routes/jobs/job.py

```diff
@@ -0,0 +1,79 @@
+"""Jobs represent the steps performed during asset creation.
+
+You can view a single tasks or all tasks associated with as specific
+asset. Only _service accounts_ can create or update tasks.
```

Comment:
Can you fix this header comment (which I think is taken from tasks) to be more accurate about jobs - maybe just 

Jobs are long-running OTF queries on a list of geometries.